config/clusters/2i2c/jackeddy.values.yaml

```diff
@@ -31,14 +31,6 @@ basehub:
             url: ""
             custom_html: <a href="https://science.nasa.gov/heliophysics/programs/living-with-a-star/">NASA's Living with a Star program</a> and <a href="https://cpaess.ucar.edu/">UCAR/CPAESS</a>
     singleuser:
-      extraFiles:
-        # FIXME: document this feature
```

Comment:
Can we open an issue and link it here? I think FIXME notes in code like this are far too hidden for them to be properly prioritised and actioned

Reply:
@sgibson91 I was not 100% sure what exactly the 'this' refers to, but based on https://github.com/2i2c-org/infrastructure/pull/3255#issue-1937880888 it's clear it's in reference to showing hidden files! So I've opened  https://github.com/2i2c-org/infrastructure/issues/3285. Thanks for the catch.